src/envs/repeated_nextto_painting.py

```diff
@@ -107,7 +109,11 @@ def _transition_move(self, state: State, action: Action) -> State:
     @property
     def predicates(self) -> Set[Predicate]:
         return super().predicates | {
-            self._NextTo, self._NextToBox, self._NextToShelf, self._NextToTable
+            self._NextTo,
+            self._NextToBox,
+            self._NextToShelf,
+            self._NextToTable,
+            self._NotOnTable,
```

Comment:
`NotOnTable` doesn't feel specific to `rnt_painting`, I think we should just make it be part of regular painting as well.